docs(api): Robot motor control in 8.6.0 (#19262)
OpenTrons/opentrons-api: api/docs/v2/advanced_control/jupyter.rst

    labware_2.set_offset(x=0.10, y=0.20, z=0.30)

    labware_3 = protocol.load_labware("nest_96_wellplate_200ul_flat", location="3")
    labware_3.set_offset(x=0.10, y=0.20, z=0.30)
    
This automatically generated code uses generic names for the loaded labware. If you want to match the labware names already in your protocol, change the labware names to match your original code:

.. code-block:: python

    reservoir = protocol.load_labware("nest_12_reservoir_15ml", "2")
    reservoir.set_offset(x=0.10, y=0.20, z=0.30)
    
.. versionadded:: 2.12

Once you've executed this code in Jupyter Notebook, all subsequent positional calculations for this reservoir in slot 2 will be adjusted 0.1 mm to the right, 0.2 mm to the back, and 0.3 mm up.

Keep in mind that ``set_offset()`` commands will override any labware offsets set by running Labware Position Check in the Opentrons App. In Flex protocols, you should only  reuse labware offset measurements when they apply to the *same labware type* used across the Flex deck. On the OT-2, use Labware Position Check to create and reuse labware offsets only for the *same labware type* in the *same deck slot* on the *same robot*.

.. warning::

> 	Improperly reusing offset data may cause your robot to move to an unexpected position or crash against labware, which can lead to incorrect protocol execution or damage your equipment. When in doubt: run Labware Position Check again and update your code!

Reviewer: Consider removing the colon (which is fine) for a more conversational tone.

```suggestion
	Improperly reusing offset data may cause your robot to move to an unexpected position or crash against labware, which can lead to incorrect protocol execution or damage your equipment. When in doubt, run Labware Position Check again and update your code!
```

        labware=plate, new_location="3"
    )  # plate now has offset 0, 0, 0
    plate.set_offset(
        x=-0.1, y=-0.2, z=-0.3
    )  # plate again has offset -0.1, -0.2, -0.3

Using Custom Labware
--------------------

If you have custom labware definitions you want to use with Jupyter, make a new directory called ``labware`` in Jupyter and put the definitions there. These definitions will be available when you call :py:meth:`~opentrons.protocol_api.ProtocolContext.load_labware`.

Using Modules
-------------

If your protocol uses :ref:`modules <new_modules>`, you need to take additional steps to make sure that Jupyter Notebook doesn't send commands that conflict with the robot server. Sending commands to modules while the robot server is running will likely cause errors, and the module commands may not execute as expected.

To disable the robot server, open a Jupyter terminal session by going to **New > Terminal** and run ``systemctl stop opentrons-robot-server``. Then you can run code from cells in your notebook as usual. When you are done using Jupyter Notebook, you should restart the robot server with ``systemctl start opentrons-robot-server``.

.. note::

>     While the robot server is stopped, the robot will display as unavailable in the Opentrons App. If you need to control the robot or its attached modules through the app, you need to restart the robot server and wait for the robot to appear as available in the app.

Reviewer: Does the word "robot" seem over-used here? Could try this:

```suggestion
    While the robot server is stopped, it appears unavailable in the Opentrons App. To control the robot or its attached modules through the app, you need to restart the server and wait for it to become available again.
```

Reviewer 2: Nope, "robot server" is a term of art and we should keep it.
